<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenLockerKcmForm</class>
 <widget class="QWidget" name="ScreenLockerKcmForm">
  <property name="geometry">
   <rect>
    <width>500</width>
    <height>525</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Activation</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_1">
       <property name="formAlignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_1">
         <property name="text">
          <string>Lock screen:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_1">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="kcfg_Autolock">
           <property name="text">
            <string>Automatically after:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPluralHandlingSpinBox" name="kcfg_Timeout">
           <property name="specialValueText">
            <string notr="true"/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="kcfg_LockOnResume">
         <property name="toolTip">
          <string>Lock screen when waking up from suspension</string>
         </property>
         <property name="text">
          <string>After waking from sleep</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Allow unlocking without password for:</string>
         </property>
         <property name="buddy">
          <cstring>kcfg_LockGrace</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KPluralHandlingSpinBox" name="kcfg_LockGrace">
         <property name="specialValueText">
          <string>Immediately</string>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="toolTip">
          <string>The global keyboard shortcut to lock the screen.</string>
         </property>
         <property name="text">
          <string>Keyboard shortcut:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="KKeySequenceWidget" name="lockscreenShortcut" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Appearance</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QQuickWidget" name="lnfConfigWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="resizeMode">
          <enum>QQuickWidget::SizeViewToRootObject</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_1">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::FieldsStayAtSizeHint</enum>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Wallpaper &amp;type:</string>
             </property>
             <property name="buddy">
              <cstring>wallpaperCombo</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="wallpaperCombo"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QQuickWidget" name="wallpaperConfigWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="resizeMode">
            <enum>QQuickWidget::SizeRootObjectToView</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>KKeySequenceWidget</class>
   <extends>QWidget</extends>
   <header>kkeysequencewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPluralHandlingSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kpluralhandlingspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_Autolock</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_Timeout</receiver>
   <slot>setEnabled(bool)</slot>
  </connection>
 </connections>
</ui>
